osu.Game.Rulesets.Mania/UI/Components/EvenColumn.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osuTK.Graphics;
+
+namespace osu.Game.Rulesets.Mania.UI.Components
+{
+    public class EvenColumn : ColumnType
```

Comment:
`ColumnTypeEven` might be better.

Reply:
As far as I've seen, derived class name mostly comes before inherited class name, sometimes in-between and can be shortened if not misleading:
    - `FooType : Type`
    - `DrawableFooType : DrawableType`